website/static/js/registrationUtils.js

```diff
@@ -339,7 +344,7 @@ var Draft = function(params, metaSchema) {
             $.each(schema.pages, function(i, page) {
                 $.each(page.questions, function(qid, question) {
                     var q = self.schemaData[qid];
-                    if (q && (q.value || '').trim()) {
+                    if (q && (q.value || '').trim() !== '') {
```

Comment:
1. No, single select questions also set a question's value attribute to a string
2. This was done for consistency with the other two calls of the same nature present in the branch.